### PR TITLE
Support AWS China conformance testing on `giraffe`

### PR DESCRIPTION
**Pull request by stone-z**
Closes https://github.com/giantswarm/giantswarm/issues/17919

Adds the `/test aws-china` trigger and associated `aws-china` Tekton resources to run AWS tests in China.

### CODE REVIEW

**Comment by stone-z** [comment]
Testing here https://github.com/giantswarm/releases/pull/754

File: tekton/tasks/cleanup.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
     description: Cluster information is stored here.
   steps:
   - name: cleanup
-    image: quay.io/giantswarm/standup:2.7.0
+    image: quay.io/giantswarm/standup:2.7.0-f6cbffe2c7f94b247320f16da95f55f20f380cb3
```
Comment (stone-z) [inline tekton/tasks/cleanup.yaml:21]: Will use tagged version once released

File: prow/configs.yaml [hunk 2]
```diff
@@ -115,8 +115,33 @@ presubmits:
               resources:
                 requests:
                   storage: 10Mi
-    trigger: "/test aws"
+    trigger: "/test aws$"
```
Comment (stone-z) [inline prow/configs.yaml:118]: Prevents running `aws-china` when only running `/test aws`. Can revert this if firecracker would like to always run both

**Comment by stone-z** [comment]
@giantswarm/team-ludacris PTAL after https://github.com/giantswarm/standup/pull/129

**Review by MarcelMue — APPROVED** [review]
LGTM after standup is tagged

**Comment by stone-z** [comment]
@giantswarm/team-firecracker-engineers FYI, after this you'll be able to run conformance tests on `giraffe` by running `/test aws-china`.

A few questions for you:
1. if you _want_ to also run in China every time you say `/test aws` (so two tests, `giraffe` + `gaia` from one command), let me know. Right now I have made them completely separate
2. `awscnfm` runs very rapidly in China. The default test plan finishes in ~20 seconds, which seems too fast, but I don't think it's something related to test-infra or standup, since the release gets created properly there. I suspect you may need to make some modifications
3. The current AWS tests don't clean up the created releases. You can use the `cleanup` Tekton task like Azure does if you'd like, otherwise would you please double check that whatever cleanup logic `awscnfm` has is working properly?

**Comment by stone-z** [comment]
Tested the `aws`, `aws-china`, and `cncf` tests using these changes. `cis` runs but has an unrelated failure I'll address in a second PR

**Pull request merged**